src/lib/formatter/SarifFormatter.ts

```diff
@@ -104,9 +105,8 @@ abstract class SarifFormatter {
 	 * Multiple RuleResult objects will be consolidated into a single run object.
 	 * https://docs.oasis-open.org/sarif/sarif/v2.0/csprd02/sarif-v2.0-csprd02.html#_Toc10127675
 	 */
-	public format(catalog: RuleCatalog, ruleResults: RuleResult[]): unknown {
+	public format(catalog: RuleCatalog, ruleResults: RuleResult[]): Run {
 		const runJson = deepCopy(this.jsonTemplate);
-		const results = runJson.results;
```

Comment:
The `results` property was superfluous and made the code harder to understand, so I removed it. We just access `runJson.results` now.